ops.py

```diff
@@ -130,30 +131,43 @@ def get_weights(self, dtype=torch.float16):
         bias = self.get_weight(self.bias, dtype)
         return (weight, bias)
 
+class GGMLOpsLayer(GGMLLayer):

```

Comment:
This is a nice way of doing it. Think they can just be part of `GGMLLayer` though, don't think that class will ever be used for anything that's not just, well, that.